File: Facturas/Models/Factura.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Facturas.Models
{
    public class Factura
    {
        public int NumeroFactura { get; set; }
        public int ClienteID { get; set; }
        public DateTime Fecha { get; set; }
        public List<LineaFactura> LineasFactura { get; set; }

        public Cliente Cliente { get; set; }


    }
}

File: Facturas_Angular/Facturas/src/app/API/models/factura.ts
/* tslint:disable */
/* eslint-disable */
import { Cliente } from './cliente';
import { LineaFactura } from './linea-factura';
export interface Factura {
  cliente?: Cliente;
  clienteID?: number;
  fecha?: string;
  lineasFactura?: null | Array<LineaFactura>;
  numeroFactura?: number;
  totalFactura?: number;
}
